README.md

```diff
@@ -463,4 +463,47 @@ def ==(other)
   other.is_a?(Quantity) && ...
   # ...or, sometimes, duck type
   other.respond_to?(:amount) && other.respond_to?(:unit) && ...
+end
+```
+
+## Pattern Matching
+
+Value objects are a prime candidate for use with the pattern matching syntax introduced in Ruby 2.7.
+
+The easiest way to integrate with pattern matching is to return a hash of attributes from `#decontruct_keys`, which you have probably already implemented in `#to_h`. This allows the object to be pattern matched using the hash syntax:
+
+```ruby
+class Quantity
+  ...
+
+  def decontruct_keys(_keys)
+    to_h
+  end
```

Comment:
As the pattern-matching feature is new and there probably will be lot of people not yet familiar with it, I feel like we should not skip through the possibility of actually using `keys` arg, and when it might be useful?

Reply:
I've added a short section on when you would use the argument.